kwalitee/cli/check.py

```diff
@@ -185,8 +186,11 @@ def _ensure_directory(filename):
 
     def _format_errors(args):
         filename, errors = args
-        return error_template.format(filename=filename, errors='\n'.join(
-            errors if len(errors) else no_errors))
+        if errors is False:
```

Comment:
The `check_file()` normally returns 2 kinds of result:
 - `errors = [ ]` interpreted as `no error`
 - `errors = [...]` (not empty array) interpreted as `there is at least a error`

But, I needed a third state to say: `the file is excluded`.
So, the easy way was return a `errors = False` value.

The `if not errors` is `True` also for empty array and I can't distinguish the case `no error` from the case `the file is excluded`.